<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameAdderDialog</class>
 <widget class="QDialog" name="GameAdderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Adder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Images/Leaner24.png</normaloff>:/Images/Leaner24.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Game Information</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="imgThumbnail">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,2,2,2">
            <item>
             <widget class="QLabel" name="lblGameName">
              <property name="text">
               <string>Game Name: N/A</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblTitleID">
              <property name="text">
               <string>Title ID: N/A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblGamerscore">
              <property name="text">
               <string>Gamerscore: N/A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblAvatarAwards">
              <property name="text">
               <string>Avatar Awards: N/A</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="treeWidgetAllGames">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>100</number>
         </attribute>
         <column>
          <property name="text">
           <string>Game Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Gamerscore</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Award Count</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="txtSearch">
             <property name="placeholderText">
              <string>Search...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnShowAll">
             <property name="maximumSize">
              <size>
               <width>41</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/undo.png</normaloff>:/Images/undo.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnShowAll_2">
             <property name="text">
              <string>Awards Only</string>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/trophy.png</normaloff>:/Images/trophy.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="treeWidgetQueue">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>100</number>
         </attribute>
         <column>
          <property name="text">
           <string>Game Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Gamerscore</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Award Count</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Add Games</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/Images/add.png</normaloff>:/Images/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/Images/cancel.png</normaloff>:/Images/cancel.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
